src/wallet/wallet.cpp

```diff
@@ -3659,15 +3659,11 @@ void CWallet::MarkReserveKeysAsUsed(int64_t keypool_id)
             m_pool_key_to_index.erase(keypool.vchPubKey.GetID());
         }
         walletdb.ErasePool(index);
+        LogPrintf("keypool index %d removed\n", index);
```

Comment:
I'd prefer to put this in the wallet debug category - it's just an info message, and not very useful for most end-users I guess?